.github/workflows/rust.yml

```diff
@@ -16,22 +16,21 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: rustup component add clippy rustfmt
-      - name: Clippy
-        run: cargo clippy --no-default-features -- -D warnings
-      - name: Clippy (alter-table)
-        run: cargo clippy --no-default-features --features alter-table -- -D warnings
-      - name: Clippy (index)
-        run: cargo clippy --no-default-features --features index -- -D warnings
-      - name: Clippy (alter-table & index)
-        run: cargo clippy --no-default-features --features "alter-table index" -- -D warnings
-      - name: Clippy (alter-table & transaction)
-        run: cargo clippy --no-default-features --features "alter-table transaction" -- -D warnings
-      - name: Clippy (index & transaction)
-        run: cargo clippy --no-default-features --features "index transaction" -- -D warnings
-      - name: Clippy (alter-table & index & transaction)
-        run: cargo clippy --no-default-features --features "alter-table index transaction" -- -D warnings
-      - name: Clippy (all)
-        run: cargo clippy --all-features --all-targets -- -D warnings
+      - name: Clippy (root)
+        run: cargo clippy -- -D warnings
+      - name: Clippy (core)
+        run: |
+          cd core
```

Comment:
<https://doc.rust-lang.org/cargo/reference/workspaces.html#package-selection>

It seems fine to run the core-related command through package selection.

Reply:
Oh, is there a command for running clippy by only picking specific package?
It looks that clippy does not have
e.g. `cargo clippy -h`
![image](https://user-images.githubusercontent.com/2025065/170814569-ea53c1c3-70ec-4c89-950e-2d5131c24e9c.png)

Reply:
I thought something like
```
cargo clippy --workspace -p gluesql-core --no-default-features
```
but clippy runs for all the workspaces.